src/resources/views/crud/list.blade.php

```diff
@@ -151,9 +151,6 @@ class="{{ backpack_theme_config('classes.table') ?? 'table table-striped table-h
 
 @section('after_styles')
   {{-- DATA TABLES --}}
```

Comment:
I'm bringing sexy back!!

![sexy-justin-timberlake-gif](https://github.com/user-attachments/assets/cd9d03bf-c066-4241-8aed-6c423840aa56)

YEAH!

```suggestion
  {{-- DATA TABLES --}}
    @basset('https://cdn.datatables.net/2.1.8/css/dataTables.bootstrap5.min.css')
    @basset("https://cdn.datatables.net/responsive/3.0.3/css/responsive.dataTables.min.css")
    @basset('https://cdn.datatables.net/fixedheader/4.0.1/css/fixedHeader.dataTables.min.css')
```